app/views/liquids/_golang.html.erb

```diff
@@ -0,0 +1,7 @@
+<iframe
+  title="golang editor"
+  src="<%= url %>"
+  height="400"
+  frameborder="0"
```

Comment:
Some browsers [support lazy loading of iframes](https://web.dev/iframe-lazy-loading/) so might as well improve loading performance on those browsers.

```suggestion
  frameborder="0"
  loading="lazy"
```